src/com/xilinx/rapidwright/edif/EDIFTools.java

```diff
@@ -773,7 +765,11 @@ public static EDIFNetlist readEdifFile(Path edifFileName){
 			}
 			edif.setEncryptedCells(new ArrayList<>(Arrays.asList(ednFiles)));
 		}
-		
+		if(ENABLE_EDIF_BINARY_CACHING) {
+		    Path bedif = edifFileName.getParent().resolve(
+                    edifFileName.getFileName().toString().replace(".edf", ".bedf"));
+		    BinaryEDIFWriter.writeBinaryEDIF(bedif, edif);
```

Comment:
Can we fail more gracefully if we don't have write permission? This would make the flag a lot safer to enable all the time.

Reply:
Made some changes so that in the face of a failed write/read of the binary EDIF, a message is printed but process continues.  In the case of reading, it will fall back to just reading the EDIF file.